backend/lib/models/Thread.js

```diff
@@ -0,0 +1,70 @@
+// -- Imports
+const { Schema, model, ObjectId } = require("mongoose");
+
+const CONVERSATION_STATUS_OPTIONS = [
+  "accepted", "archived", "blocked", "pending"
+];
+
+// -- Schema
+const participantSchema = new Schema(
+  {
+    id: {
+      ref: "User",
+      required: true,
+      type: ObjectId,
+    },
+    location: locationSchema,
```

Comment:
Looking at the designs I don't see us needing participant's `location` and I doubt we will ever need to reference it within the messaging feature itself. So I would omit here.

Reply:
I agree, I think we can omit this for all the user references here.